File: Testing tech/Python-code/find_duplicates.py
# dictionaries_sets/find_duplicates.py
lst = [1,2,3,2,4,3]
seen = set()
duplicates = set()
for x in lst:
    if x in seen:
        duplicates.add(x)
    else:
        seen.add(x)
print("Duplicates:", duplicates)

File: Testing tech/interviewCoding_Java/FindDuplicates.java
// hashmaps_sets/FindDuplicates.java
import java.util.*;
public class FindDuplicates {
    public static void main(String[] args){
        int[] arr = {1,2,3,2,4,3};
        Set<Integer> uniques = new HashSet<>();
        Set<Integer> duplicates = new HashSet<>();
        Arrays.stream(arr).forEach(n -> {
            if(!uniques.add(n)) duplicates.add(n);
        });
        System.out.println("Duplicates: " + duplicates);
    }
}
